mods/fire/init.lua

```diff
@@ -272,6 +211,7 @@ else -- Fire enabled
 			if minetest.find_node_near(pos, 1, {"group:puts_out_fire"}) then
 				return
 			end
+			minetest.sound_play("fire_small", {pos = pos, gain = 0.2})
```

Comment:
No, it makes sure that a sound is played regularly at this fire node, irregardless of whether a new sound is played at a new place or not.